adapters/Vungle/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 ## Vungle iOS Mediation Adapter Changelog
 
+#### Version 6.7.0.0
+- Verified compatibility with Vungle SDK 6.7.0.
+- Added support for playing multi ads at the same time.
+- Added support for new Vungle SDK event handlers.
```

Comment:
This should be more actionable for publishers. Other than being compatible with 6.7.0, I don't think publishers care about event handlers or init API.

Things that I think publishers will care more about:

- Clicks now reported when the click happens instead of when the ad is closed.
- Banner and interstitial ads now forward the `willLeaveApplication` callback.

Reply:
@ericleich Hi Eric, what you said makes much sense. I will remove `Added support for new Vungle SDK event handlers` and `Added support for new Vungle SDK init API`, then I will add your suggestions. Thanks for the instructions. 😊

Reply:
@ericleich Updated, thanks. 🙂